conf/application.conf

```diff
@@ -188,3 +188,9 @@ microservice {
   }
 }
 
+api {
```

Comment:
this config is not actually not needed anymore - it was there to support the user info api sutff but that was taken out